python/tk_framework_desktopserver/certificates.py

```diff
@@ -106,6 +106,7 @@ def _check_call(self, ctx, cmd):
 
         # Do not use popen.check_call because it won't redirect stderr to stdout properly
         # and it can't close stdin which causes issues in certain configurations on Windows.
+        self._logger.debug("%s: %s" % (ctx.capitalize(), cmd))
```

Comment:
Why capitalize?  I just got burned with a spurious capitalize call elsewhere... is the actual string so bad?  If not, lets leave it pure.